src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -115,7 +123,7 @@ class PaymentMethodList extends Component {
         // Hide any billing cards that are not P2P debit cards for now because you cannot make them your default method, or delete them
         const filteredCardList = _.filter(this.props.cardList, card => card.additionalData.isP2PDebitCard);
 
-        let combinedPaymentMethods = PaymentUtils.formatPaymentMethods(this.props.bankAccountList, filteredCardList, this.props.payPalMeUsername, this.props.userWallet);
+        let combinedPaymentMethods = PaymentUtils.formatPaymentMethods(this.props.bankAccountList, filteredCardList, this.props.personalBankAccount, this.props.payPalMeUsername, this.props.userWallet);
```

Comment:
Could probably just split the params into separate lines in order to shorten this line 🤔

Reply:
thanks, for some reaosn I didn't think you could do that for function arguments